### DISCUSSION TITLE
Hosting a Lemmy server?

### DISCUSSION THREAD

**Discussion by samuk** [alexnathanson/solar-protocol#89] [Ideas]
I'm interested in the possibilities for hosting dynamic content. I was thinking a [Lemmy instance ](https://slrpnk.net/) might be fun.

If there was; 

* A minimum [2Ghz, 8Gig RAM ](https://forum.banana-pi.org/t/banana-pi-bpi-cm5-computer-module-with-amlogic-a311d2-4g-lpddr4-ram-and-16g-emmc-onboard/15578)for participating servers.
* A minimum solar system spec of
 [LiFePO4wered/Pi+ solar charger / HAT](https://lifepo4wered.com/lifepo4wered-pi+.html)
[50 Ah LiFePO4 Battery (1 cell)](https://shop.gwl.eu/LiFePO4-cells-3-2-V/Winston-Cells/ThunderSky-WINSTON-TSWB-LYP50AHA-LiFeYPO4-3-3V-50Ah.html)
[50W Solar panel](https://www.aliexpress.com/item/1005005534431689.html) 
* A minimum bandwith spec of some number

And you could get 10x of these systems in Southern Europe/ Africa/ North America and 10x of these systems in Oceania.

Could you do a thing whereby there was one server at any one time that had login enabled, where users could post content. The remaining ~9 servers were acting as a budget CDN just serving static content to non logged in users.

When they plan to do a change from the 'active' server posting is disabled for the duration of a Rsync run. Once the Rsync is completed the new 'active' server enables user login.

I dunno if Lemmy supports [Webauthn](https://youtu.be/zJPNuORkvvk?t=46) but that would be nice

**Comment by poVoq** [comment]
I wrote down some comments of this idea here: https://slrpnk.net/post/353569
